assets/cubyz/biomes/ocean/shallows.zig.zon

```diff
@@ -1,53 +1,49 @@
 .{
 	.properties = .{
-		.cold,
 		.ocean,
+		.wet,
 	},
-	.type = .arctic_ocean,
-	.minHeight = -100,
-	.maxHeight = -22,
-	.radius = 500,
-	
-	.roughness = 20,
-	.hills = 10,
-
+	.radius = 128,
+	.minHeight = -64,
+	.maxHeight = -32,
```

Comment:
The shallow biome is kind of difficult to see in the rest of the ocean, maybe it should be shallower?